src/wallets/apple-pay/ApplePayButton.tsx

```diff
@@ -1,21 +1,20 @@
-import React, { PropsWithChildren } from 'react';
-import { Button } from 'react-native';
-import { ApplePayButtonStyleProps } from '../../@types/style-types';
+import React from 'react';
+import { requireNativeComponent, UIManager } from 'react-native';
+import { ApplePayButtonStyleProps, ApplePayButtonNativeProps } from '../../@types/style-types';
 
-type ApplePayButtonProps = PropsWithChildren<{
-  title: string;
-  styles: ApplePayButtonStyleProps;
-  onSubmit: () => void;
-}>;
+type ApplePayButtonProps = {
+  buttonStyles?: ApplePayButtonStyleProps;
+};
+
+const ApplePayButtonComponent = UIManager.getViewManagerConfig('ApplePayButton') !== null
```

Comment:
do we need this check? I remember trying on my test branch (which is. based on your branch), removing this does not affect the rendering result

Reply:
the test is there just in case someone accidentally modify the name. it will alert the dev working on it.
the name matches now but if anyone changes it and forgets to update there it won't match then.
plus, it is a good practice to test for null if the method can return null, we should never expect that it won't